validate-devcontainer.sh

```
@@ -0,0 +1,70 @@
+fi
+
+if [ ! -f ".devcontainer/Dockerfile" ]; then
+    echo "❌ .devcontainer/Dockerfile not found"
+    exit 1
+fi
+
+echo "✅ DevContainer configuration files found"
+
+# Test building the DevContainer
+echo "🔨 Testing DevContainer build..."
+cd .devcontainer
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)